packages/spec-core/package.json

```diff
@@ -51,7 +51,13 @@
     "xml2js": "^0.5.0",
     "yargs": "~17.7.2",
     "jackspeak": "2.1.1",
-    "multer": "^1.4.5-lts.1"
+    "multer": "^1.4.5-lts.1",
+    "cross-env": "~7.0.3",
```

Comment:
related to other comment that dep should be able to go

Reply:
cross-env is gone.